### ISSUE TITLE
support cwd in debug

### ISSUE THREAD

**Issue by ctaggart** [ctaggart/golang-vscode#7]
`cwd` doesn't currently work when I wish to do:

``` json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch",
            "type": "go",
            "request": "launch",
            "mode": "debug",
            "remotePath": "",
            "port": 2345,
            "host": "127.0.0.1",
            "program": "${workspaceRoot}",
            "env": {},
            "args": [],
            "cwd": "/Users/cameron/go/golang-vscode"
        }
    ]
}
```

Delve support is in progress https://github.com/derekparker/delve/issues/295 as https://github.com/Microsoft/vscode-go/issues/27 mentions.

**Comment by ctaggart** [comment]
I created a pull request, but don't know how to do the darwin support:
https://github.com/derekparker/delve/pull/521

**Comment by mikegleasonjr** [comment]
I think it is merged in go-delve/delve#650, any way we can go ahead in vs code?
